.github/workflows/github_release.yml

```diff
@@ -59,16 +58,44 @@ jobs:
           reno report --no-show-source --ignore-cache --version v${{ steps.version.outputs.current_release }} -o relnotes.rst
 
       - name: Convert to Markdown
-        uses: docker://pandoc/core:3.1
+        uses: docker://pandoc/core:3.8
         with:
-          args: "--from rst --to markdown_github --no-highlight relnotes.rst -o relnotes.md --wrap=none"
+          args: "--from rst --to gfm --syntax-highlighting=none --wrap=none relnotes.rst -o relnotes.md"
+
+      - name: Copy relnotes file
+        run: |
+          cat relnotes.md > enhanced_relnotes.md
+
+      - name: Add contributor list
+          # only for minor releases and minor release candidates (not bugfix releases)
+        if: endsWith(steps.version.outputs.current_release, '.0')
+        env:
+          GH_TOKEN: ${{ github.token }}
+          START: v${{ steps.version.outputs.current_release }}-rc0
+          END: ${{ github.ref_name }}
+        run: |
+          JQ_EXPR='[.commits[].author.login]
+            | map(select(. != null and . != "HaystackBot" and . != "dependabot[bot]"))
+            | unique
+            | sort_by(ascii_downcase)
+            | map("@\(.)")
+            | join(", ")'
+          CONTRIBUTORS=$(gh api "repos/deepset-ai/haystack/compare/$START...$END" \
```

Comment:
What about adding a `CONTRIBUTORS=$(...) || echo "Unable to fetch contributors"`? This way if this command fails we'll have an error message

Reply:
Done in https://github.com/deepset-ai/haystack/pull/10074/commits/02fb962510fc6f36a510fd490d61480a058edfce